packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable no-use-before-define */
+import harden from '@agoric/harden';
+import produceIssuer from '@agoric/ertp';
+import { makeZoeHelpers } from './helpers/zoeHelpers';
+
+/*
+  Roles in the arrangement:
+  - Contract creator: describes the contract with:
+    - number of seats, show, date/time of start
+    - expected (ERTP) amount per ticket (we assume all tickets cost the same)
+  - Smart Contract: 
+    - mints the tickets
+    - provides the seats
+  - Auditorium (unique contract seat, usually taken by the contract creator): the person hosting 
+  the Opera show, selling the tickets and getting the payment back
+  - Ticket buyers (contract seat created on demand): 
+    - can see the available opera show seats
+    - can consult the terms
+    - can redeem the zoe invite with the proper payment to get the ticket back
+
+  ERTP and Zoe are considered to be the most highly trusted pieces of code by everyone
+  They are more trusted than the code of this contract
+  As a consequence, they are going to be leveraged as much as possible by this contract
+  to increase its trustworthiness and by the contract users
+  
+*/
+
+export const makeContract = harden(zoe => {
+  // Create the internal ticket mint
+  const { issuer, mint, amountMath: ticketAmountMath } = produceIssuer(
+    'Opera tickets',
+    'set',
+  );
+
+  const {
+    terms: { show, start, count },
+    issuerKeywordRecord: { Money: moneyIssuer },
+  } = zoe.getInstanceRecord();
+
+  const moneyAmountMath = moneyIssuer.getAmountMath();
```

Comment:
```suggestion
  const { amountMath: moneyAmountMath } = zoe.getIssuerRecord(moneyIssuer);
```
`moneyIssuer` will be a remote object, so it will need to be accessed by `E(moneyIssuer)` which returns a promise. Instead, you can use `zoe.getAmountMaths` to get the amountMath for the moneyIssuer synchronously.

Reply:
This doesn't look fixed yet. Unresolving. Or am I misunderstanding the review process flow we're using? Or did you just forget to push?

Reply:
it isn't fixed, my mistake
I thought i had accepted the suggestion, but i hadn't